### DISCUSSION TITLE
新文档已上线，欢迎修正查错

### DISCUSSION THREAD

**Discussion by am-abudu** [Qexo/Qexo#248] [Announcements]
[Qexo 文档](http://www.oplog.cn/qexo/)

**Comment by Fgaoxing** [comment]
@am-abudu 无法访问，不过我可以提供域名！！

**Reply by am-abudu** [reply]
Azure的可访问性有点差，我尝试一些别的平台

**Reply by am-abudu** [reply]
似乎是eu.org受到阻断

亦可使用 qexo.pages.dev 访问

**Comment by Fgaoxing** [comment]
https://qexo.ga/
https://qexo.gq/

**Comment by Fgaoxing** [comment]
我提前区申请免费域名了

**Comment by Fgaoxing** [comment]
@am-abudu
